src/commandbar.js

```diff
@@ -129,106 +129,110 @@ export class CommandBar {
         if (!selection.isCollapsed || !selection.rangeCount) return;
 
         if (event.key === '/' && !this._active) {
-            this.options.onActivate && this.options.onActivate();
+            this.open(this.options.commands, event);
+        }
+    }
+
+    open(commands, initialEvent = undefined) {
```

Comment:
1. Receiving a event makes no sense, you only use the target property of the event
2. I would rename the variable `initialTarget` to `openOnKeyupTarget` or `onlyOnKeyupTarget`
3. I would pass an option dictionnary for the commands: 
```js
open()
open({commands: commands})
open({openOnKeyupTarget: target})
```